README.md

```diff
@@ -187,30 +187,44 @@ This is an example of what a re-implementation looks like:
 
 If a track implements an exercise for which test data exists, the exercise _must_ contain a `.meta/tests.toml` file. The goal of this file is to keep track of which tests are implemented by the exercise. Tests in this file are identified by their UUID and each test has a boolean value that indicates if it is implemented by that exercise.
 
-A `tests.toml` file for a track's `two-fer` exercise looks like this:
+A `tests.toml` file for a track's `two-fer` exercise might contain:
 
 ```toml
-[canonical-tests]
+[19709124-b82e-4e86-a722-9e5c5ebf3952]
+description = "no name given"
 
-# no name given
-"19709124-b82e-4e86-a722-9e5c5ebf3952" = true
+[3451eebd-123f-4256-b667-7b109affce32]
+description = "a name given"
+include = false
 
-# a name given
-"3451eebd-123f-4256-b667-7b109affce32" = true
-
-# another name given
-"653611c6-be9f-4935-ab42-978e25fe9a10" = false
+[653611c6-be9f-4935-ab42-978e25fe9a10]
+description = "another name given"
 ```
 
-In this case, the track has chosen to implement two of the three available tests.
+In this case, the track has chosen to implement two of the three available tests (`include = true` is the default and is omitted).
 
 If a track uses a _test generator_ to generate an exercise's test suite, it _must_ use the contents of the `tests.toml` file to determine which tests to include in the generated test suite.
 
 ### Track Test Data Tooling
 
-To make it easy to keep the `tests.toml` up to date, tracks can use the [`canonical_data_syncer` application](https://github.com/exercism/canonical-data-syncer). This application is a small, standalone binary that will compare the tests specified in the `tests.toml` files against the tests that are defined in the exercise's canonical data. It then interactively gives the maintainer the option to include or exclude test cases that are currently missing, updating the `tests.toml` file accordingly.
+To make it easy to keep the `tests.toml` up to date, tracks can use the [`configlet` application](https://github.com/exercism/configlet)'s `sync` command.
```

Comment:
```suggestion
To make it easy to keep the `tests.toml` up to date, tracks should use the [`configlet` application](https://github.com/exercism/configlet)'s `sync` command.
```

Should we encourage this behavior?  So "should" instead of "can"?

Reply:
I think we should (pun intended)! @ee7?